.travis.yml

```diff
@@ -6,7 +6,7 @@ android:
     - tools
     - platform-tools
     - extra-android-m2repository
-    - android-27
+    - android-28
```

Comment:
Hmm we should nuke all of these and just echo the license SHA. This block is not incremental and specifying all of these only slows down builds.

Reply:
Not sure I follow?

Reply:
https://github.com/JakeWharton/RxBinding/blob/master/.travis.yml#L7-L9

Reply:
TIL!